cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -383,64 +551,13 @@ protected void Application_BeginRequest()
 }
 ```
 
-## A6 Sensitive data exposure
-
-DO NOT: Store encrypted passwords.
-
-DO: Use a strong hash to store password credentials. Use Argon2, PBKDF2, BCrypt or SCrypt with at least 8000 iterations and a strong key.
-
-DO: Enforce passwords with a minimum complexity that will survive a dictionary attack i.e. longer passwords that use the full character set (numbers, symbols and letters) to increase the entropy.
-
-DO: Use a strong encryption routine such as AES-256 where personally identifiable data needs to be restored to it's original format. Do not encrypt passwords. Protect encryption keys more than any other asset. Apply the following test: Would you be happy leaving the data on a spreadsheet on a bus for everyone to read. Assume the attacker can get direct access to your database and protect it accordingly.
-
-DO: Use TLS 1.2 for your entire site. Get a free certificate [LetsEncrypt.org](https://letsencrypt.org/).
-
-DO NOT: [Allow SSL, this is now obsolete](https://github.com/ssllabs/research/wiki/SSL-and-TLS-Deployment-Best-Practices#22-use-secure-protocols).
-
-DO: Have a strong TLS policy (see [SSL Best Practises](http://www.ssllabs.com/projects/best-practises/)), use TLS 1.2 wherever possible. Then check the configuration using [SSL Test](https://www.ssllabs.com/ssltest/) or [TestSSL](https://testssl.sh/).
-
-DO: Ensure headers are not disclosing information about your application. See [HttpHeaders.cs](https://github.com/johnstaveley/SecurityEssentials/blob/master/SecurityEssentials/Core/HttpHeaders.cs) , [Dionach StripHeaders](https://github.com/Dionach/StripHeaders/) or disable via `web.config`:
-
-```xml
-<system.web>
-    <httpRuntime enableVersionHeader="false"/>
-</system.web>
-<system.webServer>
-    <security>
-        <requestFiltering removeServerHeader="true" />
-    </security>
-    <httpProtocol>
-        <customHeaders>
-            <add name="X-Content-Type-Options" value="nosniff" />
-            <add name="X-Frame-Options" value="DENY" />
-            <add name="X-Permitted-Cross-Domain-Policies" value="master-only"/>
-            <add name="X-XSS-Protection" value="1; mode=block"/>
-            <remove name="X-Powered-By"/>
-        </customHeaders>
-    </httpProtocol>
-</system.webServer>    
-```
-
-## A7 Missing function-level access control
+e.g Startup.cs in the Configure()
 
-DO: Authorize users on all externally facing endpoints. The .NET framework has many ways to authorize a user, use them at method level:
-
-```csharp
-[Authorize(Roles = "Admin")]
-[HttpGet]
-public ActionResult Index(int page = 1)
-```
-
-or better yet, at controller level:
-
-```csharp
-[Authorize]
-public class UserController
+``` csharp
+  app.UseHttpsRedirection();
 ```
 
-You can also check roles in code using identity features in .net: `System.Web.Security.Roles.IsUserInRole(userName, roleName)`
-
-# A8 Cross-site request forgery
+### Cross-site request forgery
```

Comment:
It miss the info link to the [CSRF CS](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md) like for the LDAP injection.